### DISCUSSION TITLE
Sync to MongoDB

### DISCUSSION THREAD

**Discussion by xamrex** [NightscoutFoundation/xDrip#2710] [Q&A]
Hello
I d like to upload  data to mongoDatabase.
I have found that mongo URI should look like this:
mongodb://{user}:{password}@{host}.mongolab.com:{11111}/{database}

I have created database at mongodb.com and here is my adress:
mongodb+srv://{USERNAME}:{PASS}@cluster0.awlvdo9.mongodb.net/?retryWrites=true&w=majority

What address should I put into URI to upload result to mongo DB?

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
@xamrex the Atlas database is not compatible with mongo upload https://xdrip.readthedocs.io/en/latest/use/cloud/#mongodb this will only work if you have a Mongo database like Railway, Northflank or directly managed in a VPS or within a container.

**Comment by szantos** [comment]


**Reply by Navid200** [reply]
Would you please open a new thread because this is a different subject?
And I will tag the person who can give you more detail if needed.

**Reply by Navid200** [reply]
On a second thought, what would be the point of this?
One of the main advantages of Nightscout on Google Cloud is that it is not limited to a 500MB database size.  If you keep your database on Atlas, you will have that limitation.
Why not use other existing solutions that are set up with that (database on Atlas) intended?

**Reply by szantos** [reply]
Thanks! Can be deleted. Comment moved to:
https://github.com/NightscoutFoundation/xDrip/discussions/2781#discussion-5092275

**Comment by tzachi-dar** [comment]
@xamrex Do you want to upload data from xdrip to nightscout? Is this your question?

**Reply by xamrex** [reply]
I have done it. I m using Northflank

**Reply by Navid200** [reply]
@xamrex You have posted here in discussions.  I take it you have a question.  
The person who can help you has asked you to clarify your question.  Instead of taking advantage of his interest and clarifying, you say "I have done it".
Then, why did you post here?  What are you hoping to achieve?
He has no reason to try to help you if you are not interested.

**Comment by xamrex** [comment]
@Navid200 
So My question at the begging was how to upload data to mongo DB. -> i wanted to create server that will download results form this Database...
But much easiest solution already existed-> nitghtscout. so 
I get the response that i should use Northflank and  now I have running nightscout server ;)
